libs/core/src/components/pds-image/test/pds-image.spec.tsx

```diff
@@ -8,7 +8,7 @@ describe('pds-image', () => {
       html: `<pds-image class="pds-image" src="//source.unsplash.com/100x100"></pds-image>`,
     });
     expect(page.root).toEqualHtml(`
-      <pds-image class="pds-image" src="//source.unsplash.com/100x100">
```

Comment:
Although the tool removed the actual prop, it would be nice to test that the `inheritedImgSpecificAttrs` works as expected.